src/components/Settings/Subscribe/Subscribe.component.js

```diff
@@ -153,13 +153,16 @@ const Subscribe = ({
   };
 
   const renderSubscriptionStatus = () => {
-    let productStatus = PROCCESING;
     const { androidSubscriptionState, expiryDate, error } = subscription;
 
-    const ERROR = 'error';
+    let productStatus = NOT_SUBSCRIBED;
 
     if (isAndroid()) {
-      productStatus = error.showError ? ERROR : androidSubscriptionState;
+      productStatus = error.showError
+        ? ERROR
+        : products.length
+        ? androidSubscriptionState
+        : EMPTY_PRODUCT;
```

Comment:
Create a function to assign the product state value. This is a bit confusing and you can avoid the use of let.